conanfile.py

```diff
@@ -50,13 +53,19 @@ def source(self):
         extracted_dir = self.name + "-" + self.version
         os.rename(extracted_dir, self.source_subfolder)
 
+        if self.version == "2.4.0":
```

Comment:
as this recipe is only for 2.4.0, I think it's not required checking the version here.